contributing-guides/style-guide.md

```diff
@@ -4,40 +4,102 @@ This page lists specific formatting instructions for `tldr` pages.
 
 ## Layout
 
-The basic format of each page should match the following template:
+The basic format of each page should match the following template and have at most 8 command examples:
 
-```
-# command-name
+```md
+# command_name
```

Comment:
For the actual pages, we use a space, so why not here?
```suggestion
# command name
```

Reply:
Isn't the title supposed to match the command's name?

Reply:
Sure, but we do write `# git commit`, instead of `# git_commit`. But actually, this change doesn't really matter.

Reply:
* `git-commit.md` becomes `# git commit` but yes.